<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Finite Element Analysis</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QTreeWidget" name="treeWidget">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="animated">
        <bool>true</bool>
       </property>
       <attribute name="headerVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="headerHighlightSections">
        <bool>true</bool>
       </attribute>
       <attribute name="headerMinimumSectionSize">
        <number>30</number>
       </attribute>
       <column>
        <property name="text">
         <string notr="true">1</string>
        </property>
       </column>
      </widget>
      <widget class="GraphicWindow" name="openGLWidget">
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="focusPolicy">
        <enum>Qt::StrongFocus</enum>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1600</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Open"/>
    <addaction name="action_Save"/>
    <addaction name="actionSave_As"/>
    <addaction name="separator"/>
    <addaction name="action_Exit"/>
   </widget>
   <widget class="QMenu" name="menu_Solver">
    <property name="title">
     <string>&amp;Solver</string>
    </property>
    <addaction name="action_Solver"/>
   </widget>
   <widget class="QMenu" name="menu_View">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="actionTopView"/>
    <addaction name="actionBottomView"/>
    <addaction name="actionRightView"/>
    <addaction name="actionLeftView"/>
    <addaction name="actionFrontView"/>
    <addaction name="actionBackView"/>
    <addaction name="actionIsoView"/>
   </widget>
   <widget class="QMenu" name="menu_Reports">
    <property name="title">
     <string>&amp;Reports</string>
    </property>
    <addaction name="actionNodesReport"/>
    <addaction name="actionElementsReport"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Solver"/>
   <addaction name="menu_View"/>
   <addaction name="menu_Reports"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_Open"/>
   <addaction name="action_Save"/>
   <addaction name="actionSave_As"/>
   <addaction name="separator"/>
   <addaction name="actiontakeSnapshoot"/>
   <addaction name="separator"/>
   <addaction name="action_Solver"/>
   <addaction name="actionSelect"/>
   <addaction name="actionLegend"/>
   <addaction name="actionTurnOffLighting"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="viewsToolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionTopView"/>
   <addaction name="actionBottomView"/>
   <addaction name="actionFrontView"/>
   <addaction name="actionBackView"/>
   <addaction name="actionLeftView"/>
   <addaction name="actionRightView"/>
   <addaction name="actionIsoView"/>
   <addaction name="separator"/>
   <addaction name="actionOrthographicProjection"/>
   <addaction name="actionPerpectiveProjection"/>
  </widget>
  <widget class="QToolBar" name="simulationToolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionstartSimulation"/>
   <addaction name="actionpauseSimulation"/>
   <addaction name="actionstopSimulation"/>
  </widget>
  <widget class="QToolBar" name="meshToolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionGrid"/>
   <addaction name="actionInitialMesh"/>
   <addaction name="actionSolved_Mesh"/>
   <addaction name="actionShowMesh"/>
  </widget>
  <widget class="QToolBar" name="resultsToolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>true</bool>
   </attribute>
   <addaction name="actionstressxx"/>
   <addaction name="actionstressyy"/>
   <addaction name="actionstresszz"/>
   <addaction name="actionstressxy"/>
   <addaction name="actionstressyz"/>
   <addaction name="actionstresszx"/>
   <addaction name="actionstressvonmises"/>
   <addaction name="separator"/>
   <addaction name="actionresultux"/>
   <addaction name="actionresultuy"/>
   <addaction name="actionresultuz"/>
  </widget>
  <action name="actionTopView">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/topview.ico</normaloff>:/icons/topview.ico</iconset>
   </property>
   <property name="text">
    <string>Top View</string>
   </property>
   <property name="toolTip">
    <string>set top view</string>
   </property>
  </action>
  <action name="actionRightView">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/rightview.ico</normaloff>:/icons/rightview.ico</iconset>
   </property>
   <property name="text">
    <string>Right View</string>
   </property>
   <property name="toolTip">
    <string>set right view</string>
   </property>
  </action>
  <action name="actionFrontView">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/frontview.ico</normaloff>:/icons/frontview.ico</iconset>
   </property>
   <property name="text">
    <string>Front View</string>
   </property>
   <property name="toolTip">
    <string>set front view</string>
   </property>
  </action>
  <action name="action_Open">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/open2.ico</normaloff>:/icons/open2.ico</iconset>
   </property>
   <property name="text">
    <string>&amp;Open</string>
   </property>
   <property name="toolTip">
    <string>open file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/save.ico</normaloff>:/icons/save.ico</iconset>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="toolTip">
    <string>save file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/saveas.ico</normaloff>:/icons/saveas.ico</iconset>
   </property>
   <property name="text">
    <string>Save &amp;As</string>
   </property>
   <property name="toolTip">
    <string>save file as</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="action_Exit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
   <property name="toolTip">
    <string>exit</string>
   </property>
   <property name="shortcut">
    <string>Alt+F4</string>
   </property>
  </action>
  <action name="action_Solver">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/solver.png</normaloff>:/icons/solver.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Solver</string>
   </property>
   <property name="toolTip">
    <string>solver the FEA model</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionstartSimulation">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/play.ico</normaloff>:/icons/play.ico</iconset>
   </property>
   <property name="text">
    <string>startSimulation</string>
   </property>
   <property name="toolTip">
    <string>start simulation</string>
   </property>
  </action>
  <action name="actionpauseSimulation">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/pause.ico</normaloff>:/icons/pause.ico</iconset>
   </property>
   <property name="text">
    <string>pauseSimulation</string>
   </property>
   <property name="toolTip">
    <string>pause simulation</string>
   </property>
  </action>
  <action name="actionstopSimulation">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/stop.ico</normaloff>:/icons/stop.ico</iconset>
   </property>
   <property name="text">
    <string>stopSimulation</string>
   </property>
   <property name="toolTip">
    <string>stop simulation</string>
   </property>
  </action>
  <action name="actiontakeSnapshoot">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/saveimage.ico</normaloff>:/icons/saveimage.ico</iconset>
   </property>
   <property name="text">
    <string>takeSnapshoot</string>
   </property>
   <property name="toolTip">
    <string>take a picture from drawing screen</string>
   </property>
  </action>
  <action name="actionBottomView">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/bottomview.ico</normaloff>:/icons/bottomview.ico</iconset>
   </property>
   <property name="text">
    <string>Bottom View</string>
   </property>
   <property name="toolTip">
    <string>set bottom view</string>
   </property>
  </action>
  <action name="actionLeftView">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/leftview.ico</normaloff>:/icons/leftview.ico</iconset>
   </property>
   <property name="text">
    <string>Left View</string>
   </property>
   <property name="toolTip">
    <string>set left view</string>
   </property>
  </action>
  <action name="actionBackView">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/backview.ico</normaloff>:/icons/backview.ico</iconset>
   </property>
   <property name="text">
    <string>Back View</string>
   </property>
   <property name="toolTip">
    <string>set back view</string>
   </property>
  </action>
  <action name="actionOrthographicProjection">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/orthographic.ico</normaloff>:/icons/orthographic.ico</iconset>
   </property>
   <property name="text">
    <string>Orthographic Projection</string>
   </property>
   <property name="toolTip">
    <string>set orthographic projection</string>
   </property>
  </action>
  <action name="actionPerpectiveProjection">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/perspective.ico</normaloff>:/icons/perspective.ico</iconset>
   </property>
   <property name="text">
    <string>Perpective Projection</string>
   </property>
   <property name="toolTip">
    <string>set perpective projection</string>
   </property>
  </action>
  <action name="actionSelect">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/select.ico</normaloff>:/icons/select.ico</iconset>
   </property>
   <property name="text">
    <string>Select</string>
   </property>
   <property name="toolTip">
    <string>select a element</string>
   </property>
  </action>
  <action name="actionGrid">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/axis_grid.ico</normaloff>:/icons/axis_grid.ico</iconset>
   </property>
   <property name="text">
    <string>Grid</string>
   </property>
   <property name="toolTip">
    <string>draw grid</string>
   </property>
  </action>
  <action name="actionInitialMesh">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/mesh_dots.png</normaloff>:/icons/mesh_dots.png</iconset>
   </property>
   <property name="text">
    <string>Initial Mesh</string>
   </property>
   <property name="toolTip">
    <string>initial mesh</string>
   </property>
  </action>
  <action name="actionSolved_Mesh">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/mesh_def.png</normaloff>:/icons/mesh_def.png</iconset>
   </property>
   <property name="text">
    <string>Solved Mesh</string>
   </property>
   <property name="toolTip">
    <string>solved mesh</string>
   </property>
  </action>
  <action name="actionstressyy">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>σyy</string>
   </property>
   <property name="toolTip">
    <string>stress yy</string>
   </property>
  </action>
  <action name="actionstresszz">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>σzz</string>
   </property>
   <property name="toolTip">
    <string>stress zz</string>
   </property>
  </action>
  <action name="actionstressxy">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>τxy</string>
   </property>
   <property name="toolTip">
    <string>stress xy</string>
   </property>
  </action>
  <action name="actionstressyz">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>τyz</string>
   </property>
   <property name="toolTip">
    <string>stress yz</string>
   </property>
  </action>
  <action name="actionstresszx">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>τzx</string>
   </property>
   <property name="toolTip">
    <string>stress zx</string>
   </property>
  </action>
  <action name="actionstressxx">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>σxx</string>
   </property>
   <property name="toolTip">
    <string>stress xx</string>
   </property>
  </action>
  <action name="actionstressvonmises">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>σv</string>
   </property>
   <property name="toolTip">
    <string>stress von mises</string>
   </property>
  </action>
  <action name="actionresultux">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>ux</string>
   </property>
   <property name="toolTip">
    <string>result displacement x</string>
   </property>
  </action>
  <action name="actionresultuy">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>uy</string>
   </property>
   <property name="toolTip">
    <string>result displacement y</string>
   </property>
  </action>
  <action name="actionresultuz">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>uz</string>
   </property>
   <property name="toolTip">
    <string>result displacement z</string>
   </property>
  </action>
  <action name="actionLegend">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/legend.png</normaloff>:/icons/legend.png</iconset>
   </property>
   <property name="text">
    <string>Legend</string>
   </property>
   <property name="toolTip">
    <string>show legend</string>
   </property>
  </action>
  <action name="actionNodesReport">
   <property name="text">
    <string>Nodes</string>
   </property>
   <property name="toolTip">
    <string>report from nodes</string>
   </property>
  </action>
  <action name="actionElementsReport">
   <property name="text">
    <string>Elements</string>
   </property>
   <property name="toolTip">
    <string>report from elements</string>
   </property>
  </action>
  <action name="actionIsoView">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/isometric.png</normaloff>:/icons/isometric.png</iconset>
   </property>
   <property name="text">
    <string>Isometric View</string>
   </property>
   <property name="toolTip">
    <string>set isometric view</string>
   </property>
  </action>
  <action name="actionTurnOffLighting">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/lightoff.ico</normaloff>
     <normalon>:/icons/lighton.ico</normalon>:/icons/lightoff.ico</iconset>
   </property>
   <property name="text">
    <string>Turn Off Lighting</string>
   </property>
   <property name="toolTip">
    <string>turn lighting</string>
   </property>
  </action>
  <action name="actionShowMesh">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/surface.ico</normaloff>:/icons/surface.ico</iconset>
   </property>
   <property name="text">
    <string>Show Mesh</string>
   </property>
   <property name="toolTip">
    <string>show mesh</string>
   </property>
  </action>
  <action name="actionShowNumbering">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/surface.ico</normaloff>:/icons/surface.ico</iconset>
   </property>
   <property name="text">
    <string>Show numbering</string>
   </property>
   <property name="toolTip">
    <string>show numbering</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>GraphicWindow</class>
   <extends>QOpenGLWidget</extends>
   <header location="global">graphicwindow.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
